src/EnergyPlus/IceThermalStorage.cc

```diff
@@ -178,13 +179,25 @@ namespace IceThermalStorage {
         return nullptr; // LCOV_EXCL_LINE
     }
 
-    void SimpleIceStorageData::simulate(const PlantLocation &EP_UNUSED(calledFromLocation),
+    void SimpleIceStorageData::simulate(const PlantLocation &calledFromLocation,
                                         bool EP_UNUSED(FirstHVACIteration),
                                         Real64 &EP_UNUSED(CurLoad),
                                         bool RunFlag)
     {
         std::string const RoutineName("SimpleIceStorageData::simulate");
 
+        // this was happening in PlantLoopEquip before
```

Comment:
@Myoldmopar the failures are happening here after being called from SimulateWaterHeaterStandAlone since PlantLoop has not been allocated yet.

Reply:
OK.  I knew that was too easy to just work right away.  Actually as I was doing this, I was noticing that RunFlag wasn't even used, so I'm not so sure this code is necessary.  I'll do further testing this morning and get it working.  Thanks!